packages/icon/package.json

```diff
@@ -4,12 +4,22 @@
   "description": "LeafyGreen UI Kit Icons",
   "main": "./dist/umd/index.js",
   "module": "./dist/esm/index.js",
-  "types": "./dist/types/src/index.d.ts",
-  "typesVersions": {},
+  "types": "./dist/types/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/esm/index.js",
+      "require": "./dist/umd/index.js",
+      "types": "./dist/types/index.d.ts"
+    },
+    "./dist/*": {
```

Comment:
As a refresher from yesterday's dev sync, this is what ensures backwards compatibility when using "dist/*" imports?

Reply:
correct, yeah